spectacles/runner.py

```diff
@@ -33,16 +33,24 @@ def is_commit(string: str) -> bool:
 
 
 class LookerBranchManager:
-    def __init__(self, client: LookerClient, project: str, remote_reset: bool = False):
+    def __init__(
+        self,
+        client: LookerClient,
+        project: str,
+        remote_reset: bool = False,
+        import_refs: Dict[str, str] = {},
+    ):
         """Context manager for Git branch checkout, creation, and deletion."""
         logger.debug(f"Setting up branch manager in project '{project}'")
         self.client = client
         self.project = project
         self.remote_reset = remote_reset
+        self.import_refs = import_refs
 
         state: ProjectState = self.get_project_state()
         self.workspace: str = state.workspace
         self.history: List[ProjectState] = [state]
+        # TODO: Should this call happen after we've checked out the correct state?
         self.imports: List[str] = self.get_project_imports()
```

Comment:
@joshtemple Is it possible this call is happening too early? We do it when we instantiate the LookerBranchManager, which is typically on production at that point. 

If someone is working on a branch where they have added a local import, we won't catch it when we call it then. 

I think it's possible that we need to make this call _after_ we've landed in the target ref state.